CHANGELOG.adoc

```diff
@@ -12,6 +12,9 @@
 
 === Bug fixes
 
+- https://github.com/eclipse-syson/syson/issues/1602[#1602] [diagrams] Fix an issue where the drag and drop of a RequirementUsage (or any other Element) with a declared short name but no declared name from the Explorer view to a diagram was not possible.
```

Comment:
You forgot to put _Explorer_ in italic and `RequirementUsage` / `Element` as code.
You can see the changelog and/or the release notes to see which words should be in italic or code.